mirage/serializers/contributor.js

```diff
@@ -0,0 +1,23 @@
+import config from 'ember-get-config';
+import ApplicationSerializer from './application';
+
+const { OSF: { apiUrl } } = config;
+
+export default ApplicationSerializer.extend({
+    links(model) {
+        return {
+            users: {
+                related: {
+                    href: `${apiUrl}/v2/users/${model.users.id}/`,
```

Comment:
Still weird that `users` is plural.

Reply:
Yeah, I'm sorry about that. It was almost certainly my fault.